src/elements/Accordion/Accordion.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * Accordion
+ */
+
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+import Div from '../Div/Div';
+import Grid from '../Grid/Grid';
+import { isMobile } from '../../util/helpers';
+
+class Accordion extends Component {
+	static propTypes = {
+		children: PropTypes.node.isRequired,
+		title: PropTypes.node.isRequired,
+		className: PropTypes.string,
+		expanded: PropTypes.bool,
+		notification: PropTypes.bool // this is used to style accordion as a notification
+	};
+
+	constructor(props) {
+		super(props);
+
+		if (this.props.expanded) {
+			this.state = {
+				showContent: true
+			};
+		} else {
+			this.state = {
+				showContent: false
+			};
+		}
+
+		this.toggleAccordion = this.toggleAccordion.bind(this);
+	}
+
+	toggleAccordion() {
+		this.setState({
+			showContent: !this.state.showContent
+		});
+	}
+
+	render() {
+		const { title, children, className, notification } = this.props,
+			{ showContent } = this.state,
+			itemWidths = showContent && !isMobile() && notification ? [3, 7, 2] : [10, 2],
+			toggleText = showContent ? 'hide' : 'show',
+			toggleIconClass = showContent ? 'arrow arrow-up' : 'arrow arrow-down';
+
+		const combinedClasses = classNames(
+			'ui-accordion',
+			this.props.notification && 'notification',
```

Comment:
We are going to start using the object notation with `classNames` now. So instead of:
```js
'ui-accordion',
this.props.notification && 'notification',
...
```

it'll be:

```js
'ui-accordion',
...,
{
 'notification': this.props.notification,
 ...
}
```

Look at the Modal.js component for a real example.